components/Loading.vue

```diff
@@ -22,7 +22,7 @@ export default {
   <div v-if="loading">
     <div class="overlay"></div>
     <div class="content" :class="{ 'content-content-mode' : mode === 'content', 'content-main-mode' : mode === 'main' }">
-      Loading...
+      <t k="principal.loading" />
```

Comment:
This should get a separate translation entry, `principal.loading` is used when a principal (small user icon + their name box like in add members to things) is being loaded inline on the page and you probably wouldn't expect changing that to affect every page or vice versa.

It's not particularly important for this usage, but `&hellip;` produces a proper "…" vs "..."; they are one symbol instead of 3 and slightly closer together.  You just need the translation to be output raw without html escaping to use it since it's an entity.

Reply:
Lauren made that change, I am agree, it should be separate one.